express-delete/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "express-delete",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.17.1"
+  }
+}
```

Comment:
The `package.json` in your `express-delete` exercise folder, however, is perfectly fine.